### ISSUE TITLE
Keyboard focus is not visible on "yes/Like" button icon

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#4938]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Keyboard only user will face difficulty if keyboard focus is not visible on "yes/Like" button icon and "No/Dislike" button icon. Keyboard users cannot understand where the focus is and will get confused.

### What browsers and screen readers do this issue affect?

Windows: Chrome with NVDA

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Keyboard focus indicator

### What is the public URL for the website?

https://servicesuat.microsoft.com/#/opdashboard/

### How to reproduce the issue?

1.	Hit the URL:  https://servicesuat.microsoft.com/#/opdashboard/ log in with v-id.
2.	TAB till "Iris bot" button in header and press ENTER key.
3.	Tab till top question for my role message and press ENTER focus will be on "Top question for my role" button Press ENTER.
4.	Press TAB key two times than press down arrow key focus will be on "here are the top question" message and press ENTER.
5.	Focus will be on "Where can I download ....." button press ENTER key.
6.	Press TAB key two times than press down arrow key to navigate "please help me in improve" message and press ENTER.
7.	Focus will be on "yes/Like" button icon and on pressing Tab key focus will be on "No/Dislike" button icon.
8.	You will observe that keyboard focus is not visible on "yes/Like" button icon and "No/Dislike" button icon.

### What do you expect?

Keyboard focus should be visible on "yes/Like" button icon and "No/Dislike" button icon.

### What actually happened?

When we press enter key on "please help me in improve" message, keyboard focus is not visible on "yes/Like" button icon and now on pressing Tab key keyboard focus will be on "No/Dislike" button icon, but focus is not visible. 

**Environment Details:**
- Application Name: Intelligent Support
- URL:  https://servicesuat.microsoft.com/#/opdashboard/
- Chrome: Version 117.0.5938.150 (Official Build) (64-bit)
- Window Version: Win 11
- OS Build: 22621.2283

### Do you have any screenshots or recordings to repro the issue?

https://github.com/microsoft/BotFramework-WebChat/assets/117266902/3bab56fa-10d0-47a8-aba1-0ef03e13d884

### MAS reference

2.4.7 – Focus Visible

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev2;#MAS2.4.3;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by Kirank915** [comment]
Bug Copied from TCS team: [11438812](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/11438812)

**Comment by Kirank915** [comment]
@compulim Could you please provide fix ETA for this bug, as the partner team blocking the compliance for this bug.

**Comment by compulim** [comment]
I can't access the Iris Copilot. Everything I asked is just asking me to create a ticket. 🥲

From the video, looks like this upvote/downvote is an Adaptive Cards.

Please send this ticket to Adaptive Cards team for further investigation, thanks.

**Comment by compulim** [comment]
Please file the issue to Adaptive Cards team.
